src/views/v2/commerce/gateway/stripe/form.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Tickets Commerce: Checkout Buttons for Stripe
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/commerce/gateway/stripe/buttons.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @since   TBD
+ *
+ * @version TBD
+ * @var bool $must_login [Global] Whether login is required to buy tickets or not.
+ * @var bool $payment_element [Global] Whether to load the Stripe Payment Element.
+ */
+
+if ( $must_login ) {
+	return;
+}
+?>
+<form id="payment-form">
+
+	<?php if ( $payment_element ) : ?>
+		<?php $this->template( 'gateway/stripe/payment_element' ); ?>
```

Comment:
Normally what we do here is avoid the `if` logic in the parent templates. In this case the `gateway/stripe/payment-element` template will receive the `$payment_element` and we do an early bail in that template if it's `empty( $payment_element )`.

The same logic for the `card-element` template.